app/services/process_hashtags_service.rb

```diff
@@ -2,7 +2,9 @@
 
 class ProcessHashtagsService < BaseService
   def call(status, tags = [])
-    tags = status.text.scan(Tag::HASHTAG_RE).map(&:first) if status.local?
+    text = status.text
+    text << " " + status.spoiler_text unless status.spoiler_text.blank?
```

Comment:
Plausible.  Is also relatively ugly due to rusty Ruby skills.  d6f97e0 may be better?